int32 classifier_ID
int32 object_ID
---
int32 result
float32 confidence
float32[] sub_classifier_decisions
